src/pages/Proxies.tsx

```diff
@@ -92,7 +92,22 @@ export default () => {
     e.stopPropagation()
     void proxyGroupLatencyTest(groupName)
   }
-
+  const encodeSvg = (svg: string) => {
```

Comment:
What does this do? Add some comments and examples to illustrate please.

Reply:
transform an SVG into a data URI
,the function referred to `https://bl.ocks.org/jennyknuth/222825e315d45a738ed9d6e04c7a88d0`